google/cloud/vision/v1/vision.yaml

```diff
@@ -0,0 +1,27 @@
+type: google.api.Service
+config_version: 2
+name: vision.googleapis.com
+title: Google Cloud Vision API
+
+apis:
+- name: google.cloud.vision.v1.ImageAnnotator
+
+documentation:
+  summary:
+    'Integrates Google Vision features, including image labeling, face, logo,
+    and landmark detection, optical character recognition (OCR), and detection
+    of explicit content, into applications.'
+
+authentication:
+  rules:
+    - selector: '*'
+      allow_without_credential: true
```

Comment:
It seems that way: https://github.com/googleapis/googleapis/blob/master/google/api/auth.proto#L70

Do you think there's something we could do with this information in our generators?